acceptance-tests/src/main/java/io/blueocean/ath/SmartWebElement.java

```diff
@@ -171,9 +98,9 @@ public void sendKeys(CharSequence... charSequences) {
      */
     public <T> T eval(String script) {
         String js = "return (function(el,elements){" + script + "})(arguments[0],arguments[1])";
-        List<WebElement> elements = getElements();
-        WebElement el = elements.iterator().next();
-        return (T)((JavascriptExecutor)getDriver()).executeScript(script, el, elements);
+        // TODO: validate this change is correct
+        WebElement el = getElement();
+        return (T)((JavascriptExecutor)getDriver()).executeScript(script, el);
```

Comment:
@kzantow perhaps you can weigh in whether this change is appropriate.

Reply:
why change this?

Reply:
`elements` no longer exists

Reply:
Can you let me know if this change seems ok in light of the fact that there is no List from `getElements()` to pass down into `executeScript` ?

Reply:
@kzantow can you let me know if you think this change is okay given that `getElements()` no longer exists, therefore passing the elements array as the last arg isn't possible anymore

Reply:
I also don't think removing getElements() is necessary or the right change. Again, making `getElements()` support a parent `SmartWebElement` will ensure that everything is always operating on the current state of the page when it's called.